File: C++/ConditionalStatements_Advanced_Lab/WorkingHours/WorkingHours.cpp
#include <iostream>
using namespace std;

int main()
{
	int time;
	string day;

	cin >> time >> day;

	if (time >= 10 && time <= 18 && day != "Sunday") {
		cout << "open";
	}
	else {
		cout << "closed";
	}
}

File: Conditional_Statements_Advanced_Lab/working_hours.py
hour = int(input())
weekday = input()

if weekday == 'Sunday' or hour < 10 or hour > 18:
    print('closed')
elif weekday == ("Monday", "Tueday", "Wednesday",
                 "Thursday", "Friday", "Saturday") and hour >= 10 or hour <= 18:
    print('open')